doc/api/test.md

```diff
@@ -345,7 +345,7 @@ changes:
     the test is `TODO`. **Default:** `false`.
   * `timeout` {number} A number of milliseconds the test will fail after.
     If unspecified, subtests inherit this value from their parent.
-    **Default:** `30_000`.
+    **Default:** `undefined`.
```

Comment:
I think `Infinity` is a better description of the default timeout.
```suggestion
    **Default:** `Infinity`.
```

Reply:
Unfortunately, the implementation only seems to support unsigned 32-bit integers, and seems to silently ignore `Infinity` (as well as floating-point numbers). If that changes, using `Infinity` here would indeed make sense.

Reply:
That seems incorrect, the API does accept `Infinity` as value for `timeout`, and treats it as such – in fact, it treats any non-UInt32 as `Infinity`, we may want to add some validation at some point.

Reply:
Why does it imply that?

Reply:
@ljharb Strictly speaking, it does not, of course. I don't have the energy to make my point, so I'll just accept the suggestions.